src/H5FDsubfiling/mercury/src/util/CMakeLists.txt

```diff
@@ -0,0 +1,295 @@
+project(MERCURY_UTIL C)
```

Comment:
Can we separate the configure checks from the build commands?
Plus can the configure checks be in a file that can be optioned into the existing config checks files?

Reply:
In this case, I don't believe we even use this CMake file, so I'll revisit what we actually use out of Mercury.